css/panel-manager-style.css

```diff
@@ -0,0 +1,48 @@
+.panel-manager {
+  display: none;
+}
+
+#slidebar {
+  position: fixed;
+  top: 70px;
+  right: 249px;
+  z-index: 100;
+  /* transform: rotate(-90deg); */
+}
+
+#slidebar button {
+  border-radius: 4px 0 0 4px;
+  border-right: none;
+  height: 123px;
+  width: 30px;
+  /* border-right: none; */
+}
+
+#slider {
```

Comment:
The name slider is a bit too pervasive here. Maybe change to sth more specific such as sidepanel-slider.